packages/vx-axis/src/utils/center.js

```diff
@@ -1,7 +1,7 @@
 export default function center(scale) {
-  var offset = scale.bandwidth() / 2;
+  let offset = scale.bandwidth() / 2;
```

Comment:
I did not. :'( I might have had too much trust in the --fix. 

That was a dumb mistake. From now on I will always test the gallery locally!

As for the var. Don't lets work in "child" scopes, but not in above scopes?